### PR TITLE
added section about horizontal pod autoscaling

### PR DESCRIPTION
**Pull request by thecodeassassin**
* Added section about Horizontal Pod Autoscaler

### CODE REVIEW

**Comment by thecodeassassin** [comment]
Current dependencies on/for this PR:
* main
  * **PR #7** <a href="https://app.graphite.dev/github/pr/symbiosis-cloud/symbiosis-docs/7" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/symbiosis-cloud/symbiosis-docs/7?utm_source=stack-comment).

File: docs/guides/hpa.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+EOF
+```
+
+Now you can create a simple autoscaler as such:
+
+```
+kubectl autoscale deployment php-apache --cpu-percent=50 --min=1 --max=10
+```
+
+If the targets show up correctly as x%/50% then it means that the Horizontal Pod Autoscaler works as intended and the .
```
Comment (Styren) [inline docs/guides/hpa.md]: Sentence is incomplete

File: docs/guides/hpa.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+          requests:
+            cpu: 200m
+EOF
+```
+
+Now you can create a simple autoscaler as such:
+
+```
+kubectl autoscale deployment php-apache --cpu-percent=50 --min=1 --max=10
+```
```
Comment (Styren) [inline docs/guides/hpa.md:76]: Should we write a small section below this on what this means?

File: docs/guides/hpa.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
+---
+sidebar_position: 4
+---
+
+# Horizontal Pod Autoscaling
+
+This guide will help you set up a Horizontal Pod Autoscaler in your cluster.
+
+You can find more information about what Horizontal Pod Autoscaler is and how it works [here](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/).
+
```
Comment (Styren) [inline docs/guides/hpa.md:14]: Perhaps we can add just a sentence or two on what a HPA is?

File: docs/guides/hpa.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
+---
+sidebar_position: 4
+---
+
+# Horizontal Pod Autoscaler (HPA)
+
+This guide will help you set up a Horizontal Pod Autoscaler (also called HPA) in your cluster.
+
+In short, A Horizontal Pod Autoscaler will autoscale your workload (deployments, statefulsets etc) based on metrics such as CPU or Memory.
```
Comment (Styren) [inline docs/guides/hpa.md]: Just a small suggestion to clarify that it scales number of replicas.

```suggestion
In short, A Horizontal Pod Autoscaler can automatically scale the amount of replicas for your deployments, stateful sets or other workloads. Scaling can be configured based on metrics like CPU or Memory.
```

**Review by Styren — APPROVED** [review]
Just a minor suggestion, otherwise LGTM!

**Pull request merged**